extensions/wikia/ArticleVideo/scripts/ooyala-player.js

```diff
@@ -126,7 +126,18 @@ define('ooyala-player', function () {
 						tag_url: vastUrl
 					}
 				],
-				useGoogleCountdown: true
+				useGoogleAdUI: true,
+				useGoogleCountdown: false,
+				onAdRequestSuccess: function (IMAAdsManager) {
+					IMAAdsManager.addEventListener('loaded', function (eventData) {
+						var player = html5Player.player;
+
+						if (eventData.A.vpaid === true) {
```

Comment:
Most likely not. Good catch, fixed in https://github.com/Wikia/app/pull/13246/commits/2a168017e740139fe2e2f91b6cd5641c8c8eb6aa